### PR TITLE
Update quantile_regressions.do

### PR DESCRIPTION
**Pull request by mavzunat**

### CODE REVIEW

File: quantile_regressions.do [hunk 1]
```diff
@@ -8,7 +8,7 @@ append using "C:\Dropbox\BerkeleyMIDS\projects\unesco_chapter\tusome_core.dta"
 // we want to separate our demographic variables from our instrumental variables
 local demo_vars = "project language reference comparison"
 // these are the variables we'll comapre across two time points
-local vars_to_compare = "mean cv ratio_p90p10 ratio_p75p25 pct_zero gini ge2_overall ge2_ingroup ge2_outgroup between_ge2 within_ge2"
+local vars_to_compare = "mean cv ratio_p90p10 ratio_p75p25 pct_zero gini ge2_overall between_ge2 within_ge2"
```
Comment (mavzunat) [inline quantile_regressions.do:11]: These variables are empty: ge2_ingroup, ge2_outgroup. The code breaks down once they are called upon

File: quantile_regressions.do [hunk 2]
```diff
@@ -91,6 +91,8 @@ merge 1:1 dataset match_key measure_label using "`time_1'", keepusing(*_com)
 
 drop is_time*
 
+local vars_to_compare = "mean cv ratio_p90p10 ratio_p75p25 pct_zero gini ge2_overall between_ge2 within_ge2"
+
```
Comment (mavzunat) [inline quantile_regressions.do:95]: it looks like there were too many other operations between the creation of the local variable and the time when it is used in the loop. This is the only way I can get the loop to work

File: quantile_regressions.do [hunk 3]
```diff
@@ -137,6 +138,7 @@ end
 clonevar mean_line_of_equality = mean_ref
 clonevar cv_line_of_equality = cv_ref
 clonevar pct_zero_line_of_equality = pct_zero_ref
+clonevar gini_line_of_equality = gini_ref
```
Comment (mavzunat) [inline quantile_regressions.do:141]: this variables was missing entirely. I supposed that's what it should be

File: quantile_regressions.do (hunk 1 above)
Comment (TSSlade) [reply quantile_regressions.do:11]: I think they're remnants of an earlier iteration of the code in which I had misunderstood what `indeqdec0` was returning. Good eye on removing them.

File: quantile_regressions.do (hunk 2 above)
Comment (TSSlade) [reply quantile_regressions.do:95]: Not sure what you mean by that. Can you elaborate a little further? Provided everything is run in one go, the quantity of operations should not be an issue...

**Pull request merged**